sonic-xcvrd/xcvrd/dom/utilities/status/db_utils.py

```diff
@@ -0,0 +1,102 @@
+from xcvrd.dom.utilities.db.utils import DBUtils
+from xcvrd.dom.utilities.status.utils import StatusUtils
+from swsscommon import swsscommon
+
+
+class StatusDBUtils(DBUtils):
+    """
+    This class provides utility functions for managing DB operations
+    related to transceiver status.
+    """
+
+    def __init__(self, sfp_obj_dict, port_mapping, xcvr_table_helper, task_stopping_event, logger):
+        super().__init__(sfp_obj_dict, port_mapping, task_stopping_event, logger)
+        self.xcvr_table_helper = xcvr_table_helper
+        self.status_utils = StatusUtils(self.sfp_obj_dict, logger)
+        self.logger = logger
```

Comment:
@mihirpat1 do you think it would be better if logger identifier matches with the class name? Instead of being passed by the thread

Reply:
@prgeor Based on our discussion, we will keep the current implementation as it is. Including the thread name in the log message and ensuring the message's uniqueness will help identify the appropriate code.